Sources/Public/Item Views/DayOfWeekView.swift

```diff
@@ -0,0 +1,218 @@
+// Created by Bryan Keller on 9/12/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+// MARK: - DayOfWeekView
+
+/// A view that represents a day-of-the-week header in a calendar month. For example, Sun, Mon, Tue, etc.
+public final class DayOfWeekView: UIView {
+
+  // MARK: Lifecycle
+
+  public init(invariantViewProperties: InvariantViewProperties) {
+    self.invariantViewProperties = invariantViewProperties
+
+    backgroundLayer = CAShapeLayer()
+    let backgroundShapeDrawingConfig = invariantViewProperties.backgroundShapeDrawingConfig
+    backgroundLayer.fillColor = backgroundShapeDrawingConfig.fillColor.cgColor
+    backgroundLayer.strokeColor = backgroundShapeDrawingConfig.borderColor.cgColor
+    backgroundLayer.lineWidth = backgroundShapeDrawingConfig.borderWidth
+
+    label = UILabel()
+    label.font = invariantViewProperties.font
+    label.textAlignment = invariantViewProperties.textAlignment
+    label.textColor = invariantViewProperties.textColor
+
+    super.init(frame: .zero)
+
+    isUserInteractionEnabled = false
+
+    backgroundColor = invariantViewProperties.backgroundColor
+
+    layer.addSublayer(backgroundLayer)
+
+    label.translatesAutoresizingMaskIntoConstraints = true
```

Comment:
I'd remove this since we're not using constraints and this already defaults to true. Either value wouldn't affect this component.